forge-gui/res/cardsfolder/d/denry_klin_editor_in_chief.txt

```diff
@@ -4,7 +4,8 @@ Types:Legendary Creature Cat Advisor
 PT:2/2
 K:ETBReplacement:Other:CounterChoice
 SVar:CounterChoice:DB$ PutCounter | ETB$ True | CounterType$ P1P1,First Strike,Vigilance | SpellDescription$ CARDNAME enters the battlefield with your choice of a +1/+1, first strike, or vigilance counter on it.
-T:Mode$ ChangesZone | ValidCard$ Creature.nonToken+YouCtrl | Origin$ Any | Destination$ Battlefield | TriggerZones$ Battlefield | IsPresent$ Card.Self+HasCounters | Execute$ TrigPutCounters | TriggerDescription$ Whenever a nontoken creature enters the battlefield under your control, if NICKNAME has counters on it, put the same number of each kind of counter on that creature.
+T:Mode$ ChangesZone | ValidCard$ Creature.nonToken+YouCtrl | Origin$ Any | Destination$ Battlefield | TriggerZones$ Battlefield | CheckSVar$ X | Execute$ TrigPutCounters | TriggerDescription$ Whenever a nontoken creature enters the battlefield under your control, if NICKNAME has counters on it, put the same number of each kind of counter on that creature.
```

Comment:
isn't IsPresent More clean?

Reply:
Some judges said it should use LKI (so destroying it in response doesn't stop the counters)

Reply:
okay but is `CheckSVar$ Count$CardCounters.ALL`  better?